map/src/components/map-layout.tsx

```diff
@@ -22,63 +28,150 @@ interface Props {
     this.setState({ includingHidden: update.includingHidden });
 
+  private setSearchClosed = (searchClosed: boolean) => {
+    this.setState({ searchClosed });
+  };
+
+  handleChange(e: React.ChangeEvent<HTMLInputElement>) {
+    const textValue = e.target.value;
+    this.props.updateFilter('searchText', textValue);
+  }
+
+  handleSubmit(e: React.FormEvent<HTMLFormElement>) {
+    this.props.updateFilter('filterExecuted', false);
+    e.preventDefault();
+  }
+
   public render() {
-    const { className, components, page, filter, updateFilter } = this.props;
-    const { includingHidden } = this.state;
+    const { className, components, page, updateFilter } = this.props;
+    const { includingHidden, searchClosed } = this.state;
     return (
-      <div className={`${className} page-${page.page}`}>
-        <MapLoader className="map" child={components.map} />
-        <div className="overlay">
-          <div className="panel">
-            <div className="controls">
-              <div className="row">
-                <Search className="search" searchInputId="main" />
-              </div>
-              <div className="row">
-                <FilterType
-                  className="filter"
-                  filter={filter}
-                  updateFilter={updateFilter}
-                />
-                <MyLocation className="my-location" />
-              </div>
-              {includingHidden && (
-                <div className="row">
-                  <FilterVisibility
-                    className="filter"
-                    filter={filter}
-                    updateFilter={updateFilter}
-                  />
+      <AppContext.Consumer>
+        {({ lang }) => (
+          <div className={`${className} page-${page.page}`}>
+            <MapLoader className="map" child={components.map} />
+            <div className="overlay">
+              <div className="panel">
+                <div className={`controls ${searchClosed ? 'close' : ''}`}>
+                  <button
+                    type="button"
+                    className="header"
+                    onClick={() => this.setSearchClosed(!searchClosed)}
+                  >
+                    <span className="label">
+                      {t(lang, s =>
+                        searchClosed
+                          ? s.controls.openSearch
+                          : s.controls.closeSearch,
+                      )}
+                    </span>
+                    <span className="grow" />
+                    <Chevron className="toggle chevron" />
+                  </button>
+                  <form onSubmit={this.handleSubmit} className="form">
+                    <div className="row">
+                      <Search className="search" searchInputId="main" />
+                    </div>
+                    <h3 className="filter filter-heading">
+                      {t(lang, s => s.filterForm.title)}
+                    </h3>
```

Comment:
As discussed, the "Filter Markers" bar isn't necessary to help save us some extra space. 
Also, "Organization Type" and "Services Offered" can also be removed and added to the Text Box placeholder ("Select organization type" and "Select services you need"). 
![image](https://user-images.githubusercontent.com/29135917/130388199-18612d2f-7cf7-47dc-9445-959bcc7617e7.png)